.github/workflows/build_ubuntu.yml

```diff
@@ -0,0 +1,92 @@
+name: Build Ubuntu
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+
+env:
+  # Customize the CMake build type here (Release, Debug, RelWithDebInfo, etc.)
+  BUILD_TYPE: Release
+
+jobs:
+  build:
+    # The CMake configure and build commands are platform agnostic and should work equally
+    # well on Windows or Mac.  You can convert this to a matrix build if you need
+    # cross-platform coverage.
+    # See: https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions/managing-complex-workflows#using-a-build-matrix
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Info
+      run: echo GITHUB_WORKSPACE=$GITHUB_WORKSPACE
+      
+    - name: Apt update
+      run: sudo apt-get update
```

Comment:
apt-get update needs a `-y` flag to automaticly update the tree instead of waiting for user interaction.